wagtailio/project_styleguide/templates/patterns/pages/blog/blog_index_page.html

```diff
@@ -9,8 +9,41 @@
     <div class="grid">
         <section class="blog-listing">
             {% for post in posts %}
-                {% if forloop.first or forloop.counter|divisibleby:6 %}
-                    {% include 'patterns/pages/blog/includes/blog_post.html' with post=post featured=True %}
+                {# only show featured posts on the first pagination page, no filters #}
+                {% if featured_posts and posts.number == 1 and 'category' not in request.GET %}
+                    {% comment %}
+                        An even number of posts, n, is required "in between" featured posts.
+                        This is dependent on the number of featured posts.
+                        We therefore display a featured post when `forloop.counter` is odd.
+                        With a pagination of 10:
```

Comment:
suggestion: simplify the case comments so they are clear for non-developers.

I may be thick, but I find it difficult to follow the logic.

As far as I can see the only special case is when there are more featured posts than regular blog posts on the first page, which we should never reach at this point

Reply:
Thanks! Done in 5519e0f